Bump BUC version (#1441)
kyma-project/kyma: docs/service-catalog/docs/012-details-provisioning-and-binding.md

### Delete a ServiceBinding

Kyma unbinding can be achieved in two ways:

> 1. Delete the ServiceBindingUsage. The Secret injected into application will be deleted by BindingUsageController but the Secret still exist.

Reviewer: ```suggestion
1. Delete the ServiceBindingUsage. The Secret injection is deleted from the application by the BindingUsageController but the Secret still exist in the Namespace.
```
